props/saber_chris_buttons.h

```diff
@@ -0,0 +1,1036 @@
+      if (SFX_voldown) {
+        hybrid_font.PlayCommon(&SFX_voldown);
+      } else {
+        beeper.Beep(0.5, 2000);
+      }
+      STDOUT.print("Volume Down - Current Volume: ");
+      STDOUT.println(dynamic_mixer.get_volume());
+    } else {
+      #ifdef VOLUME_MENU_CYCLE
+        if (!min_vol_reached_) {
+          if (SFX_volmin) {
+            hybrid_font.PlayCommon(&SFX_volmin);
+          } else {
+            beeper.Beep(0.5, 1000);
+          }
+          STDOUT.print("Minimum Volume: ");
+          min_vol_reached_ = true;
+        } else {
+          dynamic_mixer.set_volume(VOLUME);
+          if (SFX_volmax) {
+            hybrid_font.PlayCommon(&SFX_volmax);
+          } else {
+            beeper.Beep(0.5, 3000);
+          }
+          STDOUT.print("Maximum Volume: ");
+          min_vol_reached_ = false;
+        }
+      #else
+        if (SFX_volmin) {
+          hybrid_font.PlayCommon(&SFX_volmin);
+        } else {
+          beeper.Beep(0.5, 1000);
+        }
+        STDOUT.print("Minimum Volume: ");
+      #endif
+      
+    }
+  }
+
+
+//  Button Clicker to play press/release wav files when buttons are pressed.
+//  Intended for Scavenger hilt where wheel makes tactile feel difficult.
+//  Requires press.wav and release.wav files to work.
+    void PlaySound(const char* sound) {
+        RefPtr<BufferedWavPlayer> player = GetFreeWavPlayer();
+        if (player) {
+            if (!player->PlayInCurrentDir(sound)) player->Play(sound);
+        }
+    }            
+ bool Event2(enum BUTTON button, EVENT event, uint32_t modifiers) override {      
+    switch (EVENTID(button, event, modifiers)) {
+      case EVENTID(BUTTON_POWER, EVENT_PRESSED, MODE_ANY_BUTTON | MODE_ON):
+      case EVENTID(BUTTON_POWER, EVENT_PRESSED, MODE_ANY_BUTTON | MODE_OFF):
+      case EVENTID(BUTTON_AUX, EVENT_PRESSED, MODE_ANY_BUTTON | MODE_ON):
+      case EVENTID(BUTTON_AUX, EVENT_PRESSED, MODE_ANY_BUTTON | MODE_OFF):
+        SaberBase::RequestMotion();
+        PlaySound("press.wav");
+        return false;
+      case EVENTID(BUTTON_POWER, EVENT_RELEASED, MODE_ANY_BUTTON | MODE_ON):
+      case EVENTID(BUTTON_POWER, EVENT_RELEASED, MODE_ANY_BUTTON | MODE_OFF):
+      case EVENTID(BUTTON_AUX, EVENT_RELEASED, MODE_ANY_BUTTON | MODE_ON):
+      case EVENTID(BUTTON_AUX, EVENT_RELEASED, MODE_ANY_BUTTON | MODE_OFF):
+        PlaySound("release.wav");
+        if (SaberBase::Lockup()) {
+          SaberBase::DoEndLockup();
+          SaberBase::SetLockup(SaberBase::LOCKUP_NONE);
+          return true;
+        } else {
+          return false;
+        }
+      case EVENTID(BUTTON_AUX, EVENT_PRESSED, MODE_ON):
+      case EVENTID(BUTTON_AUX2, EVENT_PRESSED, MODE_ON):
+        if (accel_.x < -0.15) {
+          pointing_down_ = true;
+        } else {
+          pointing_down_ = false;
+        }
+        return true;
+   
+
+  // Gesture Controls
+#ifdef SA22C_SWING_ON
+  case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_OFF):
```

Comment:
This thing is indented completely differently from the previous piece of code, even though they are on the same "level" and in the same function. That makes it harder for the reader to see what is going on, please use consistent indentint.

Reply:
I've been through and tidied up all the indentations. Hoping I've done it right.